src/components/services/Radarr.vue

```diff
@@ -93,11 +104,14 @@ export default {
         })
         .catch(handleError);
       if (!this.item.legacyApi) {
-        this.fetch(`${this.apiPath}/movie?apikey=${this.item.apikey}`)
-          .then((movies) => {
-            this.missing = movies.filter(
-              (m) => m.monitored && !m.hasFile
-            ).length;
+        this.fetch(`${this.apiPath}/wanted/missing?pageSize=1&apikey=${this.item.apikey}`)
+          .then((overview) => {
+			this.fetch(`${this.apiPath}/wanted/missing?pageSize=${overview.totalRecords}&apikey=${this.item.apikey}`)
```

Comment:
Not a fan of the double request to avoid handling the pagination but it doesn't seems to have any limit on the pageSize, so I guess it should be fine!

Indentation looks funny, could you rebase  your PR to get the latest changes and run `pnpm lint` ? Thanks :pray:

Reply:
Yeah I don't love the double request method, but it seemed like it would be more responsive that handling the pagination. Do you have a policy of respecting pagination in other parts of the codebase? If so then I am happy to update it to do so. 

I think I've fixed the indentation (I was using tabs instead of spaces!) but full disclosure, this is the first time I've ever done a PR on GitHub. 20+ years experience as a software engineer but I've never done a PR in Git.

Reply:
There is no specific policy regarding pagination, let keep it like that, implementing the proper pagination would be overkill.

Congrats on your first PR @Bothari :tada:! And thanks A LOT for contributing to open source. It's really appreciated, especially for Homer, I can't test and maintain all the custom cards myself because I don't use all the services (this is a good example, I don't use Radarr!).

Reply:
Thank you! I also just committed some bad code and managed to revert it. 😂 

I think Homer is a fantastic tool BTW, so congratulations on it. Really simple to use and extend. Love it.